src/app/find-help/view-offer-accommodation/view-offer-accommodation.component.html

```diff
@@ -1,7 +1,7 @@
 <app-breadcrumb></app-breadcrumb>
 
 <div class="d-flex flex-column flex-md-row" *ngIf="data">
-  <mat-card class="offer p-4 w-100 w-md-50">
+  <mat-card class="offer p-4 w-100 w-md-50 d-flex flex-column justify-content-between">
```

Comment:
Isn't w-100 more accurate here? Sorry if I am wrong.

Reply:
I am not sure what do you mean.
`w-100` for mat-card width and It isn't related to `back-link` position

Reply:
I only do not understend the need of use `justify-content-between`. I doubt if it is needed. That's all. I supposed it is maybe to make a width to 100% of a container. Does it have any other reason?

Reply:
okay, I think I got it. Check solution pls